tests/swift/tests/Tests/FlatBuffers.Test.SwiftTests/ByteBufferTests.swift

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2024 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import XCTest
+@testable import FlatBuffers
+
+final class ByteBufferTests: XCTestCase {
+  func testCopyingMemory() {
```

Comment:
Are these just sanity checks for the properties of UnsafePointers?

Reply:
Mainly yeah, making sure that we aren't copying, that we can load data properly from the pointersm for example the test testReadingNativeStructs